core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -149,7 +149,11 @@ export class ExplorationHtmlFormatterService {
       interactionChoices = interactionCustomizationArgs.choices.value.map(
         choice => choice.html);
     }
-
+    var interactionChoicesD = null;
```

Comment:
The variable name is kinda confusing. Can you make it more descriptive?

Reply:
changed